section4/exercises/classMethods.js

```diff
@@ -28,13 +28,34 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein(proteins) {
```

Comment:
Nice job with your dynamic functions using parameters! My one nit-pick is that since you are changing the protein, singular, your argument for `changeProtein` should probably just be `protein`, dropping the `s`.